net-p2p/syncthing/syncthing-0.14.12.ebuild

```diff
@@ -17,6 +17,8 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86 ~arm"
 IUSE="tools"
 
+DEPEND="tools? ( >=dev-lang/go-1.7.1:= )"
```

Comment:
Yes, `tools` is good. Just make sure to mention those examples in metadata.xml.

Besides, how are those tools used if they're in libexec? They expose some extra functionality inside syncthing?